packages/react-components/src/atoms/NavigationLink.tsx

```diff
@@ -89,6 +97,7 @@ const NavigationLink: React.FC<NavigationLinkProps> = ({
         activeClassName={activeClassName}
         css={({ colors }) => [
           styles,
+          !hasStrokeWidth && noStrokeStyles,
```

Comment:
instead of re-writting styles, we can just separate the stroke styles and apply them if the variable is `true`